### PR TITLE
Fix 80 column on newgem gemspec

### PR DESCRIPTION
**Pull request by spk**
80 column is not respected on newgem created, this have to be changed every new gem for those respecting this convention.
Cheers

### CODE REVIEW

**Comment by segiddins** [comment]
👍🏻

File: lib/bundler/templates/newgem/newgem.gemspec.tt [hunk 1]
```diff
@@ -21,10 +21,13 @@ Gem::Specification.new do |spec|
   if spec.respond_to?(:metadata)
     spec.metadata['allowed_push_host'] = "TODO: Set to 'http://mygemserver.com'"
   else
-    raise "RubyGems 2.0 or newer is required to protect against public gem pushes."
+    raise "RubyGems 2.0 or newer is required to protect against" \
```
Comment (segiddins) [inline lib/bundler/templates/newgem/newgem.gemspec.tt]: please move the space up to this line and squash the change down to a single commit? Thanks!

Comment (spk) [inline lib/bundler/templates/newgem/newgem.gemspec.tt]: Fixed o>

**Comment by segiddins** [comment]
@homu r+

**Pull request merged**